api/server.go

```diff
@@ -0,0 +1,59 @@
+// Package server provides a server object that represents the GoCert backend
+package server
+
+import (
+	"crypto/tls"
+	"net/http"
+	"time"
+
+	"github.com/canonical/gocert/internal/certificates"
+)
+
+// loadServerCertificates takes in a certificate and a private key and determines
+// whether to use self signed or given certificates, then returns it in a format
+// expected by the server
+func loadServerCertificates(certificate, key string) (*tls.Certificate, error) {
```

Comment:
```suggestion
func loadServerCertificates(certificate, key string) (tls.Certificate, error) {
```
The documentation examples do not use pointers but values, I think we should follow that convention.

Reply:
This function is gone